beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1070,32 +1066,27 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
     pub fn get_blobs(
         &self,
         block_root: &Hash256,
-        data_availability_boundary: Epoch,
     ) -> Result<Option<BlobsSidecar<T::EthSpec>>, Error> {
         match self.store.get_blobs(block_root)? {
             Some(blobs) => Ok(Some(blobs)),
             None => {
-                // Check for the corresponding block to understand whether we *should* have blobs.
```

Comment:
updated here: https://github.com/emhane/lighthouse/pull/2/commits/28702c9d5d448ede4b2d9a2673add5921541812e

Reply:
Oh man, it was updated. Scary rebase issues. Thanks for keeping an eye out for them with me!

Reply:
Wait, at glance it looks identical to the method I coded in the check dab rpc pr, that's why I recognised it. Anyhow will merge from now on to avoid these automatic rebase problems.